tests/utils/landing_state.ts

```diff
@@ -13,11 +14,14 @@ const extractLintActionFields = ({
   ...rest,
 });
 
-const extractTaskFailureFields = ({
-  detailPath: _detailPath,
-  ...rest
-}: TaskFailure): TaskFailure => ({
+const extractTaskFailureFields = (
+  sandboxPath: string,
+  { detailPath: _detailPath, ...rest }: TaskFailure
+): TaskFailure => ({
   ...rest,
+  details: _detailPath
+    ? fs.readFileSync(path.resolve(sandboxPath, _detailPath), { encoding: "utf-8" })
+    : undefined,
```

Comment:
seems like there's no reason to alias `detailPath` to `_detailPath` here since it gets used